.github/dependabot.yml

```diff
@@ -1,21 +1,5 @@
 version: 2
 updates:
-  ##### NuGet packages for Backbone.sln #####
-  - package-ecosystem: "nuget"
-    directory: "/"
-    schedule:
-      interval: "weekly"
-    groups:
-      update-nuget-dependencies:
-        patterns:
-          - "*"
-        exclude-patterns:
-          - "Npgsql.EntityFrameworkCore.PostgreSQL"
-    reviewers:
-      - "tnotheis"
-    labels:
-      - "dependencies"
-
   ##### GitHub Actions #####
```

Comment:
What are you doing??? :D :D
I was just talking about the removal of the exclude of the Npgsql package. Not about disabling the whole configuration for NuGet

Reply:
You saw nothing 🤫